### PR TITLE
Add post for PGXN Architecture presentation

### PR DESCRIPTION
**Pull request by theory**
YouTube links updated in 0aa00ab,

### CODE REVIEW

**Comment by theory** [comment]
Amplify check failure has no error messages :-(

**Comment by samay-sharma** [comment]
@DarrenBaldwin07 : Can you check why this is not building and see how easy it would be to show the error messages from amplify here?

File: docusaurus/blog/2024-02-01-pgxn-architecture/index.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+---
+slug: pgxn-architecture
+title: "Presentation: Introduction to the PGXN Architecture"
+authors: [theory]
+tags: [postgres, pgxn, extensions, architecture, presentation, video]
+# image: ./dw_social.png
+date: "2024-02-01T18:45:39Z"
```
Comment (DarrenBaldwin07) [inline docusaurus/blog/2024-02-01-pgxn-architecture/index.md:7]: Looks like this date is causing the build to fail. I think format needs to be like this `2021-09-13T18:00` - the zero timezome at the end might be what its not liking

```
2024-02-01T07:37:10.409Z [WARNING]: [ERROR] Can't format blog post date "Invalid Date"
2024-02-01T07:37:10.410Z [WARNING]: [ERROR] Processing of blog source file "2024-02-01-pgxn-architecture/index.md" failed.
                                    [ERROR] Unable to build website for locale en.
2024-02-01T07:37:10.410Z [WARNING]: [ERROR] RangeError: Invalid time value
```

**Comment by theory** [comment]
Rebased in d4547fc.

File: docusaurus/blog/2024-02-01-pgxn-architecture/index.md (hunk 1 above)
Comment (theory) [reply docusaurus/blog/2024-02-01-pgxn-architecture/index.md]: Answer:


* `docusaurus/blog/2024-01-24-data-warehouse-stack/index.md` has

    ```yaml
    date: 2024-01-25T12:00
    ```

* The [atom feed](https://tembo.io/blog/atom.xml) entry for that post has:

    ```xml
    <updated>2024-01-25T12:00:00.000Z</updated>
    ```

* Conclusion: The `date` front matter field is interpreted in UTC (at least in our config).

**Review by DarrenBaldwin07 — APPROVED** [review]

File: docusaurus/blog/2024-02-01-pgxn-architecture/index.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
+---
+slug: pgxn-architecture
+title: "Presentation: Introduction to the PGXN Architecture"
+authors: [theory]
+tags: [postgres, pgxn, extensions, architecture, presentation, video]
+# image: ./dw_social.png
```
Comment (samay-sharma) [inline docusaurus/blog/2024-02-01-pgxn-architecture/index.md:6]: Let's add an image as that image shows up in socials or when we post to twitter, linkedin, reddit etc.

File: docusaurus/blog/2024-02-01-pgxn-architecture/index.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+want to work with the Postgres community to design, build, and benefit from the
+best possible system of extension design, development, and distribution as we
+can.
+
+But as we start collaborating with the community and thinking through the [jobs
+and tools] required of the extension ecosystem of the future, we thought it
+would be useful to learn about some of the distribution architecture that exists
+today.
+
+To that end, [PGXN] creator and maintainer [David Wheeler] put together a
```
[the remark this answers is not in the document]
Comment (theory) [reply docusaurus/blog/2024-02-01-pgxn-architecture/index.md:32]: A TL;DR! Will do.

File: docusaurus/blog/2024-02-01-pgxn-architecture/index.md (hunk 2 above)
Comment (theory) [reply docusaurus/blog/2024-02-01-pgxn-architecture/index.md:6]: Hrm. I don't think I've ever made a diagram of the architecture. That might be useful.

File: docusaurus/blog/2024-02-01-pgxn-architecture/index.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (theory) [reply docusaurus/blog/2024-02-01-pgxn-architecture/index.md:32]: Along with a simple diagram, 0f95ed2 adds a brief description of each component of the architecture, to whet the reader's appetite for more!

File: docusaurus/blog/2024-02-01-pgxn-architecture/index.md (hunk 2 above)
Comment (theory) [reply docusaurus/blog/2024-02-01-pgxn-architecture/index.md:6]: Added a simple diagram in 0f95ed2. Happy to replace with something prettier if we have the design capacity and willingness.

**Comment by theory** [comment]
88c47f5 increases the size of the diagram, it was too low-res before.

**Comment by theory** [comment]
Put the image in the front matter in 210eb92 so it shows up in socials.

**Pull request merged**